media/js/firefox/os/firefox-os-new.js

```diff
@@ -0,0 +1,264 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($) {
+    'use strict';
+
+    var fxNavConfig = {
+        primaryId: 'os'
+    };
+
+    Mozilla.FxFamilyNav.init(fxNavConfig);
+
+    var $window = $(window);
+    var isSmallViewport = $window.width() < 760;
+    var $signup_content;
+    var $get_phone_content;
+
+    /*
+    * Sign up form
+    */
+    $('.newsletter-signup-toggle').on('click', function(e) {
+      e.preventDefault();
+
+      var cta = (this.id === 'signup-toggle-icon') ? 'Sign Me Up - Nav' : 'Sign Me Up - Primary';
```

Comment:
I don't see an element with an ID `signup-toggle-icon` in the template, so looks like there's only one currently.